deployability/modules/allocation/generic/models.py

```diff
@@ -59,6 +68,7 @@ class CreationPayload(InputPayload):
     label_issue: str | None = None
     label_team: str | None = None
     label_termination_date: str | None = None
+    instance_name: str | None = None
```

Comment:
As this class inherits from `InputPayload` it doesnt require to re-define the attributes (that is always that both classes require the same attribute with the same 'specs')

Reply:
Only the "new" attributes should be defined